plume-file-web-upload-jersey/README.md

```diff
@@ -14,21 +14,44 @@ Setup
   <artifactId>plume-file-web-upload-jersey</artifactId>
 </dependency>
 ```
+2. Create the webservice file with the injected dependency `FileMimeTypeDetector` (it comes from Plume file Core)
+
+Uploading with Multipart
+------------------------
+
+Using the Jersey Multipart API is the easiest way to upload a file.
+This API allows you to get the file InputStream directly from the webservice parameters, by reading the multipart content.
+
+This method preferably used for small files (< 100Mo).
+
+1. Add the Jersey Multipart to your dependencies if not already in your project
+```xml
+<dependency>
+  <groupId>org.glassfish.jersey.media</groupId>
+  <artifactId>jersey-media-multipart</artifactId>
+  <version>3.1.3</version>
+</dependency>
+```
 2. Register the MultiPart Feature on your `JerseyConfigProvider`:
 ```java
 config.register(MultiPartFeature.class);
 ```
-3. Create the webservice file with the injected dependency `FileMimeTypeDetector` (it comes from Plume file Core)
-4. Create the webservice endpoint to upload a file (using `multipart/form-data`):
+3. Create the webservice endpoint to upload a file (using `multipart/form-data`):
 ```java
 @POST
 @Operation(description = "Upload a file")
 public void upload(
     @FormDataParam("file") FormDataBodyPart fileMetadata,
-    @FormDataParam("file") InputStream fileData
+    @FormDataParam("file") InputStream fileData,
+    @FormDataParam("part-2") String part2 // an other part of your multipart request
```

Comment:
Je mettrai plutôt @FormDataParam("otherValue") String otherValue // another value present in the multipart request